src/main/java/net/sf/jabref/gui/keyboard/KeyBindingViewModel.java

```diff
@@ -0,0 +1,134 @@
+/*  Copyright (C) 2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.gui.keyboard;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyEvent;
+import org.apache.commons.lang.WordUtils;
+
+/**
+ * This class represents a view model for objects of the KeyBinding
+ * class. It has two properties representing the localized name of an
+ * action and its key bind. It can also represent a key binding category
+ * instead of a key bind itself.
+ *
+ */
+public class KeyBindingViewModel {
+
+    private KeyBinding keyBinding = null;
+    private String realBinding = "";
+    private final SimpleStringProperty name = new SimpleStringProperty();
+    private final SimpleStringProperty shownBinding = new SimpleStringProperty();
+    private final KeyBindingCategory category;
+
+
+    public KeyBindingViewModel(KeyBinding keyBinding, String binding, KeyBindingCategory category) {
```

Comment:
The third parameter is not required as far as I can see (if it's a binding, then its category is stored in it)